core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/SamExtensionsSpec.scala

```diff
@@ -0,0 +1,242 @@
+package org.broadinstitute.dsde.rawls.dataaccess
```

Comment:
unit tests in this file are copied, almost verbatim, from WorkspaceAdminServiceUnitTests.